src/Chapter2.hs

```diff
@@ -740,8 +760,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
-
+contains :: Int -> [[Int]] -> [[Int]]
+contains n = filter (\x -> elem n x)
```

Comment:
You can apply eta-reduction on lambda functions as well:
```suggestion
contains n = filter (elem n)
```

Reply:
Awesome. In love with eta-reduction. I wish more programming languages introduce this feature 😄